src/main/resources/templates/fragments/userBar.html

```diff
@@ -29,12 +23,13 @@
           </a></li>
         </ul>
       </li>
-      <li th:if="!${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}">
-        <!--  -->
+      <li th:if="!${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}"
+          th:remove="tag">
         <a class="piwik-event govuk-link" data-event-id="Sign In" href="/signin" id="user-signin">Sign
           in /
           Register</a>
       </li>
+      <li><a class="govuk-link" href="http://www.example.com">Developer hub forum</a></li>
```

Comment:
Please use the developer hub forum link

Reply:
We don't have one yet, so the example.com place holders are there to make them easy to find with valid links.

Reply:
Are we not linking it to the current dev hub forum that is live?

Reply:
In general we have not linked to old ones that we should supersede. It will make it harder to spot errors in our coding.
If the link was resourced, then we should be safer, when it's injected into the ThymeLeaf view.

Reply:
As per my understanding we are not changing the developer hub forum, it is staying the same. So we should link to the live forum.

Reply:
The Forum is on our tickets, just much lower priority than anything else, but setting it in the properties would make the change easy, if the dev-hub is removed from scope.

Reply:
Micheal was able to confirm that the current live address should be used.